db/migrations/0001_initial.py

```diff
@@ -1,44 +1,58 @@
-# Generated by Django 4.0.2 on 2022-08-19 07:39
```

Comment:
Do not modify existing migration
Add new one

Reply:
This comment is not resolved, you should create a new migration and do not modify the existing one